src/johan_stabekk_ex/ex02/message_entropy.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+__author__ = 'Johan Stabekk'
+__email__ = 'johan.stabekk@nmbu.no'
+
+
+def letter_freq(txt):
+    """
+    This code counts the frequency of how many times a character is used.
+    :param txt: input for the user of the code.
+    :return: Returns a dictionary where the frequency of characters in txt is
+            placed, with the character as a key and the number
+            of times as a value
+    """
+    freq = {}
+    txt_lowered = txt.lower()
+
+    for char in txt_lowered:
+        if char in freq.keys():
+            freq[char] += 1
+        else:
+            freq[char] = 1
+
+    return freq
+
+
+def entropy(message):
+    """ Calculates the entropy of a message according to information
+    theory.
+
+    Parameters
+    ----------
+    message - A message that the user wants to find the entropy of.
+
+    Returns - The entropy value of the message.
+    -------
+
+    """
+
+    message = letter_freq(message)
+    h = 0
+    for n_i in message.values():
+        p_i = n_i / sum(message.values())
```

Comment:
`sum(message.values())` is reused and recalculated in this loop. Would be better if you put it outside 
Also, do you know it's equal to `len(message)`, which provided less calculation here.